specificatie/openapi.yaml

```diff
@@ -558,6 +600,127 @@ components:
+        pattern: '^[0-9]{3}[1-9]|[0-9]{2}[1-9][0-9]|[0-9][1-9][0-9]{2}|[1-9][0-9]{3}$'
+        example: '2096'
+
+    geconstateerd:
+      description: 'Geef aan of je naar een geconstateerd object wilt zoeken.'
+      name: geconstateerd
+      in: query
+      required: false
+      schema:
+        type: boolean
+        default: false
+        
+    statusPand:
+      description: "Status van het pand."
+      name: statusPand
+      in: query
+      required: false
+      schema:
+        type: array
+        minItems: 1
+        items:
+          $ref: '#/components/schemas/StatusPandEnum'
+
+    bouwjaar:
+      description: >-
+        Bouwjaar van het pand.
+        Geef het minimale en/of maximale bouwjaar van het pand op waarnaar je wilt zoeken. 
+        Bv: bouwjaar[min]=1970&bouwjaar[max]=2010
+      name: bouwjaar
+      in: query
+      required: false
+      schema:
+        $ref: '#/components/schemas/IntegerFilter'
+      explode: true
+      style: deepObject
+
+    type:
+      description: "Geeft aan of het adresseerbaar object een ligplaats, standplaats of verblijfsobject is."
+      name: type
+      in: query
+      required: false
+      schema:
+        $ref: '#/components/schemas/TypeAdresseerbaarObjectEnum'
+
+    gebruiksdoelen:
+      description: "Als een verblijfsobject één van de gebruiksdoelen in deze lijst heeft, dan wordt het verblijfsobject in het zoekresultaat meegenomen."
+      name: gebruiksdoelen
+      in: query
+      required: false
+      schema:
+        type: array
+        minItems: 1
+        items:
+          $ref: '#/components/schemas/GebruiksdoelEnum'
+
+    oppervlakte:
+      name: oppervlakte
+      description: >-
+        Geef de minimale en maximale oppervlakte op waarbinnen je wilt zoeken (in m2). 
+        Bv: oppervlakte[min]=100&oppervlakte[max]=200
+      in: query
+      required: false
+      schema:
+        $ref: '#/components/schemas/IntegerFilter'
+      style: deepObject
+      explode: true
+
+    punt:
+      description: >-
+        Coordinaten van een locatie die als query-parameter gebruikt wordt om een object te zoeken. 
+        Let op, explode is true dus het formaat is ?punt[type]=Point&punt[coordinates]=194602.722,464154.308
+      name: punt
+      in: query
+      required: false
+      schema:
+        $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/pointGeoJSON.yaml'
+      style: deepObject
+      explode: true
+
+    bbox:
+      description: >-
+        Rechthoek waarbinnen het object dat je zoekt valt. 
+        Objecten die de rand van de rechthoek raken worden in het zoekresultaat meegenomen. 
+        Formaat: bbox=x lb, y lb, x ro, y ro of bbox=x lb, y lb, z lb, x ro, y ro, z ro. 
+        Daarbij zijn x lb en y lb (evt. z lb) de coördinaten van het punt linksboven en x ro en y ro (evt. z ro) de coördinaten van het punt rechtsonder.
+        De oppervlakte van de bounding box mag maximaal 250.000 m2 zijn.  
```

Comment:
is 250.000 de Nederlandse notatie? Dus 250 duizend vierkante meter?

Reply:
Aangepast